src/main/resources/templates/db_scripts/schema.json

```diff
@@ -200,6 +200,13 @@
         {
           "fieldName": "acqUnitIds"
         }
+      ],
+      "ginIndex": [
+        {
+          "fieldName": "poNumber",
+          "caseSensitive": false,
+          "removeAccents": true
+        }
```

Comment:
Resolves these 2 WARN logs:
`14:46:12 [543648/orders;980086/orders-storage] [diku] [c88a8880-f8ec-55c5-a808-14cfb0744108] [mod_orders_storage] WARN  CQL2PgJSON           Doing LIKE search without index for purchase_order.jsonb->>'poNumber', CQL >>> SQL: poNumber == 10011 >>> lower(f_unaccent(purchase_order.jsonb->>'poNumber')) LIKE lower(f_unaccent('10011'))
`


`10:34:20 [084878/orders;442135/orders-storage] [diku] [c88a8880-f8ec-55c5-a808-14cfb0744108] [mod_orders_storage] WARN  CQL2PgJSON           Doing wildcard LIKE search without index for purchase_order.jsonb->>'poNumber', CQL >>> SQL: purchaseOrder.poNumber == *10004* >>> lower(f_unaccent(purchase_order.jsonb->>'poNumber')) LIKE lower(f_unaccent('%10004%'))
`